src/App.js

```diff
@@ -1,17 +1,27 @@
 import './App.css';
+import React, { useState } from 'react';
 import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
 import Header from './components/Header.js';
 import Footer from './components/Footer.js';
 import Home from './views/Home.js';
+import ProductsList from './views/ProductsList';
 
 function App() {
   const { data, isLoading } = useFeaturedBanners();
+  const [isProductsList, setIsProductsList] = useState(false);
   console.log(data, isLoading);
+  let layout;
```

Comment:
The better approach for this is to use a `const`. As a general advise try using const as much as you can. If you are wondering how can I change this `let` to `const`? Remember that while you cannot reassign the value of a const you can create a ternary operator when initializing the value. There is nothing bad in doing it with a let and with a if but I really think it's easier to read and with less code to do a ternary operator. This section of code from line 13 to 19 could be something like this
```suggestion
const layout = isProducList ? <ProductsList /> : <Home setIsProductsList={setIsProductsList} />
```

Reply:
Commited. Also I've fixed a typo and got rid of an if in the code.